packages/bundlers/default/src/DefaultBundler.js

```diff
@@ -1911,6 +1914,17 @@ async function loadBundlerConfig(
     });
   }
 
+  if (modeConfig.unstable_manualSharedBundles) {
+    let nameArray = modeConfig.unstable_manualSharedBundles.map(a => a.name);
+    let nameSet = new Set(nameArray);
+    if (nameSet.size != nameArray.length) {
+      let err = new Error();
```

Comment:
Think this was left behind accidentally?

Reply:
I changed it to an assertion, but this was intentional, I'd like `loadConfig` to error if the `name` field of a `manualSharedBundle` is not unique.